app/implementation.py

```diff
@@ -0,0 +1,80 @@
+from .models import RedFlag
+import datetime
+import re
+
+red_flags = {}
```

Comment:
it would be better if you stored the red flags as a list instead but I need to first try out your API